cli-plugins/apps/debug-workload.hbs.md

```diff
@@ -67,6 +67,64 @@ There are known errors that will make the workload enter in an error or unknown
 
 - *TemplateRejectedByAPIServer*
-	- *Message*: Unable to apply object [ns/workload-name] for resource [source-provider] in supply chain [source-to-url]: failed to get unstructured [ns/workload-name] from api server: imagerepositories.source.apps.tanzu.vmware.com "workload-name" is forbidden: User "system:serviceaccount:ns:default" cannot get resource "imagerepositories" in API group "source.apps.tanzu.vmware.com" in the namespace "ns"
+	- *Message*: Unable to apply object `[ns/workload-name]` for resource `[source-provider]` in supply chain `[source-to-url]`: failed to get unstructured `[ns/workload-name]` from api server: imagerepositories.source.apps.tanzu.vmware.com "workload-name" is forbidden: User "system:serviceaccount:ns:default" cannot get resource "imagerepositories" in API group "source.apps.tanzu.vmware.com" in the namespace "ns"
 	- *Cause*: This error happens when the service account in the workload object does not have permissions to create objects that are stamped out by the supply chain.
 		- *Resolution*: This can be fixed by setting up the [Set up developer namespaces to use installed packages](https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.3/tap/GUID-set-up-namespaces.html) with the required service account and permissions.
+
+## <a id="steps-failure"></a> Supply Chain steps
+
+After a workload is created, the steps it goes through in the supply chain are surfaced with `tanzu apps workload get` command.
+The output looks like the following:
+
+```bash
+...
+📦 Supply Chain
+   name:   source-to-url
+
+   NAME               READY   HEALTHY   UPDATED   RESOURCE
+   source-provider    True    True      71m       gitrepositories.source.toolkit.fluxcd.io/spring-petclinic
+   image-provider     True    True      70m       images.kpack.io/spring-petclinic
+   config-provider    True    True      69m       podintents.conventions.carto.run/spring-petclinic
+   app-config         True    True      69m       configmaps/spring-petclinic
+   service-bindings   True    True      69m       configmaps/spring-petclinic-with-claims
+   api-descriptors    True    True      69m       configmaps/spring-petclinic-with-api-descriptors
+   config-writer      True    True      69m       runnables.carto.run/spring-petclinic-config-writer
+
+🚚 Delivery
+   name:   delivery-basic
+
+   NAME              READY   HEALTHY   UPDATED   RESOURCE
+   source-provider   True    True      69m       imagerepositories.source.apps.tanzu.vmware.com/spring-petclinic-delivery
+   deployer          True    True      69m       apps.kappctrl.k14s.io/spring-petclinic
+
+💬 Messages
+   No messages found.
+...
+```
+
+In the `Supply Chain` section, the resources of the supply chain related to the workload are displayed, alongside their reconcile state. In case of failure, the `READY` column will show `Unknown` or `False`, and the `HEALTHY` column will surface, as its name implies, if the resource is healthy or not.
```

Comment:
In the `Supply Chain` section, the supply chain steps associated with the workload are displayed in a table, alongside their reconcile state. In the event there's a failure for a given step, the `READY` column value will be `Unknown` or `False`, and the `HEALTHY` column value will be `False`.